rllib/utils/deprecation.py

```diff
@@ -40,9 +40,11 @@ def deprecation_warning(
     )
 
     if error:
-        if issubclass(error, Exception):
+        if not type(error) is bool and issubclass(error, Exception):
```

Comment:
@sven1977 This did not conform to the docstring before. We should be able to input a bool here and that's where all the errors came from.